### ISSUE TITLE
train_text_to_image_lora_sdxl.py validation images are completely blank

### ISSUE THREAD

**Issue by nikkolodeon** [huggingface/diffusers#5432]
### Describe the bug

I am trying to finetune SDXL with train_text_to_image_lora_sdxl.py. Unfortunately, the created validation images are all completely blank. 
The test images created at the end of the process, contain the expected output. 

### Reproduction

!accelerate launch  ./diffusers/examples/text_to_image/train_text_to_image_lora_sdxl.py \
  --pretrained_model_name_or_path="stabilityai/stable-diffusion-xl-base-1.0" \
  --train_data_dir="./dataset" --image_column="image" --caption_column="text" \
  --resolution=512 \
  --train_batch_size=1 \
  --num_train_epochs=100 --checkpointing_steps=5000 \
  --learning_rate=1e-04 --lr_scheduler="constant" --lr_warmup_steps=0 \
  --seed=42 \
  --output_dir="test" \
  --validation_prompt="Cartoon man dressed as a lawyer with a briefcase in the style of xzretoitrqq" --report_to="wandb"

### Logs

```shell
There are no logs.
```

### System Info

- `diffusers` version: 0.22.0.dev0
- Platform: Linux-5.15.120+-x86_64-with-glibc2.35
- Python version: 3.10.12
- PyTorch version (GPU?): 2.0.1+cu118 (True)
- Huggingface_hub version: 0.17.3
- Transformers version: 4.34.0
- Accelerate version: 0.23.0
- xFormers version: not installed
- Using GPU in script?: Yes
- Using distributed or parallel set-up in script?:  No

### Who can help?

@patrickvonplaten @sayakpaul

**Comment by sayakpaul** [comment]
Could you try training using a better VAE as advised here?

https://github.com/huggingface/diffusers/blob/main/examples/text_to_image/README_sdxl.md

**Comment by nikkolodeon** [comment]
Thank you very much. This worked.

**Comment by sayakpaul** [comment]
Closing it then! :)
